### ISSUE TITLE
A little confused with graphcool & graphcool-framework & prisma-cli

### ISSUE THREAD

**Issue by likeben** [graphcool/prisma#1629]
```javascript
const server = new GraphQLServer({
  typeDefs: 'src/schema.graphql',
  resolvers,
  context: req => ({
    ...req,
    // it is new Graphcool in the Docs
    db: new Prisma({
      typeDefs: 'src/generated/prisma.graphql',
      endpoint: process.env.PRISMA_ENDPOINT, // the endpoint of the Prisma DB service (value is set in .env)
      secret: process.env.PRISMA_SECRET, // taken from database/prisma.yml (value is set in .env)
      debug: true, // log all GraphQL queries & mutations
    }),
  }),
})
```
Am I miss something? Feel like the cli name change frequently, expect  provide one universal

**Comment by Boid-John** [comment]
You are not the only one who is confused. 

Recently they have announced that graphcool-framework cli will be required for the (no longer supported) graphcool framework, while the graphcool cli will support the new graphcool 1.0. 

However, this github repository was named "Framework", and was renamed to simply "Graphcool". You can see in the readme the image used still says "Graphcool Framework".  I currently use "graphcool framework" and I am still using the "graphcool" cli without issue. 

I hope they provide a very clear migration path for users. And I hope they are more transparent with their development roadmap in the future.

**Comment by kbrandwijk** [comment]
@Boid-John graphcool-framework (a.k.a. graphcool 0.x) is still supported! In the first place because 1.0 is only in the RC stage, so graphcool 0.x is still the latest final release.
Of course, 1.0 RC will reach final soon, but that does _not_ mean that graphcool 0.x will not be supported anymore. There will be a migration path, but support will _not_ be dropped.

**Comment by Boid-John** [comment]
It looks to me that all graphcool-framework example code has been
overwritten with prisma. Many links in the documentation are 404s. Users in
the forums have been encouraged to use 1.0 (now it's called prisma?)
I've been using graphcool since October and I've lost confidence in the
team. Documenting and communicating your tech needs to be a higher
priority. This is not a great way to treat your early adopters.

**Comment by ferdinandsalis** [comment]
I am confused too ☺️ The graphql space is developing quickly. I am sure the graphcool team is working hard on updating and explaining things.

**Comment by likeben** [comment]
Graphcool is really cool. But it's better to release the new version and docs when everything is ready, which will be more friendly for developers.

**Comment by sorenbs** [comment]
I'm really sorry about the confusion!

Let me try to clear things up :-)

The Graphcool Framework is a all-in-one framework that combines a GraphQL query engine (now called Prisma), serverless functions and managed hosting (Graphcool Cloud).

Prisma is a major rewrite of the query engine. You can think of Prisma as the query engine powering the Graphcool Framework.

Today Prisma can be used stand alone as a GraphQL database layer in combination with GraphQL Binding. Later we will change the Graphcool Framework to use Prisma as the query engine.

In short - if you like the features of Graphcool Framework, you can keep using it like nothing changed.

If you like to stay at the forefront of GraphQL, then check out GraphQL binding and consider using it together with Prisma.

**Comment by sorenbs** [comment]
We are currently going through all examples and documentation to make sure all links work. If you find anything that doesn't please report it here.

**Comment by Boid-John** [comment]
Why isn't Prisma in it's own repository? It looks to me like Graphcool-Framework repo was simply overwritten without a trace.

404 here:
https://www.graph.cool/docs/reference/api-gateway/overview-ucoohic9zu

Multiple 404s on this page, including a redirects to Prisma documentation and example code, telling you to install two other CLI tools on top of graphcool cli (shouldn't it acutally say graphcool-framework cli?), super confusing.
https://www.graph.cool/docs/quickstart/

This page has been unfinished for months.
https://www.graph.cool/docs/tutorials/auth/authorization-for-a-cms-miesho4goo

Basically all of the graphcool content on this website has been replaced with Prisma:
https://www.howtographql.com/

All of the graphcool-framework code examples on github are gone, replaced with primsa typescript functions.  For example, there is no official graphcool-framework code example for email-password auth available anymore.

It's hard for me to believe that Graphcool-Framework has any future at all....

**Comment by prateekrastogi** [comment]
@Boid-John As a repo watcher seating on sidelines, waiting for graphcool framework project to be matured enough for use in production, these changes came as a real shocker and confusing for me too. It seems to me that graphcool as a company has pivoted to enterprise use cases and thrown away the framework project without bothering about communicating the same appropriately.

@sorenbs  While prisma together with graphql binding and graphql yoga  looks kind of amazing in potential., the combination at this stage looks kind of inadequate from what might be expected from a proper backend framework. Can you elaborate about the ideal tech stack for backend development with these technologies? Also, can you please share more details about the evolution roadmap of graphcool framework and/or yoga+prisma+binding ?

**Comment by lucas-bremond** [comment]
@sorenbs It seems the confusion is still there...

> The Graphcool Framework is a all-in-one framework that combines a GraphQL query engine (now called Prisma) [...]

> [...] Later we will change the Graphcool Framework to use Prisma as the query engine.

Is Graphcool Framework using Prisma or not?

**Comment by marktani** [comment]
Thanks @likeben @Boid-John @lucas-bremond @kbrandwijk @ferdinandsalis @prateekrastogi for your feedback and chiming in.

I put the Graphcool Framework and Prisma into perspective [in a forum post here](https://www.graph.cool/forum/t/graphcool-framework-and-prisma/2237?u=nilan). I suggest to continue this discussion in the forum thread.

Let me know if you have any other questions.
